src/Sample.Extension/Sample.Extension.csproj

```diff
@@ -11,6 +11,8 @@
     <AssemblyName>Sample.Extension</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
This Sample extension project shouldn't need any new Mobile references. You haven't modified it, right?